# zrj: add missing link against -lm in libGNUMail.so
# NOTYPE  GLOBAL DEFAULT  UND {ceil, floorf}
# fixes x11/gnustep-app (binutils 2.27 ld.gold is unhappy)
--- Framework/GNUMail/GNUmakefile.orig	2014-10-26 23:04:28.000000000 +0200
+++ Framework/GNUMail/GNUmakefile
@@ -101,6 +101,9 @@ GNUMail_OBJC_FILES = \
 
 GNUMail_LIB_DIRS = -L../$(GNUSTEP_LIBRARIES_ROOT)
 GNUMail_GUI_LIBS = -lPantomime -lAddresses -lAddressView -lssl -lcrypto
+
+SHARED_LD_POSTFLAGS += -lm
+
 ADDITIONAL_INCLUDE_DIRS = -I.
 
 # We want to get all the warnings... 
